### DISCUSSION TITLE
Add Xbox as Predefined Site

### DISCUSSION THREAD

**Discussion by craeckor** [JulianPrieber/littlelink-custom#837] [Feature requests]
You can actually still view Xbox-profiles with https://xbox.com/play/user/{gamertag}.
Would be nice, if this gets implemented.

**Comment by tvx-matt** [comment]
I am personally going to attempt to implement this in my own instance, if it goes well, I will suggest it to the team.

**Reply by tvx-matt** [reply]
@craeckor I have made a pull request #855 to implement this. I would also personally benefit from this.

**Comment by tvx-matt** [comment]
I also think that Threads should be added, as a predefined site and a FontAwesome Icon
